spec/delayed/job_groups/job_group_spec.rb

```diff
@@ -140,6 +140,47 @@
         expect(job_group).to have_been_destroyed
       end
     end
+
+    context "on_completion_job refers to missing class" do
+      # The on_completion_job needs the class to be defined this way in order to serialize it
+      # rubocop:disable RSpec/LeakyConstantDeclaration,Style/ClassAndModuleChildren,Lint/ConstantDefinitionInBlock
+      module Delayed::JobGroups::JobGroupTestHelper
```

Comment:
Should we define this constant in a `before` so we're sure it will be present when the example starts running? The current approach won't work if we ever added multiple examples to this example group.